### DISCUSSION TITLE
Transcribe with Qualcoder?

### DISCUSSION THREAD

**Discussion by rytmisk** [ccbogel/QualCoder#609] [Q&A] [answered]
EDIT: I found the feature. Except it does not work under my OsX installation. None of the shortcuts work. I will test it on my Ubuntu-box tomorrow.
Hi
This is a phenomenal software. Reminds me a lot about Transana that is also open source, but costs more and more so Qualcoder seems to be my software of choice from now on. It also has some really interesting coding features that is not present in Transana.
Anyway: In the discussions in "Issues" here in Github some posts indicated that it was possible to do the transcription of audio/video within Qualcoder. I can not find this function in the manual or in the software. Does this mean that it does not exist? Would be great if it is possible!
All the best!
Ketil

**Comment by ccbogel** [comment]
Hi, I am glad you like QualCoder. I do not have a Mac to test the software on.
The speech to text is discussed on this page:
https://github.com/ccbogel/QualCoder/wiki/05-Files

Speech to text

QualCoder 2.9 and and newer versions a Speech to text function is included. The ffmpeg software must be installed for this to work. When an audio or video file is added to the QualCoder project an empty transcription file is created. The cogs button activates this function, but only if there is no text entered into the transcription file. First, open (view) the audio/video from Manage Files.

Speech to text allows use of several online third-party services. Most require sign up. Google does not but is limited to 50 requests of the service per day (e.g. 50 x 1 minute audio segments). QualCoder converts the audio file into 30 or 60 second chunks and adds a timestamp for each of these. Each chunk is provided to the online service which then takes a few second to convert to text.

Language options are entered into the Language box, or might be configured when you signed up to a service such as Houndify or Wit.ai. Language options are entered as text in format like: en-US, fr-FR, de-DE, es-ES, pt-PT, pt-BR.

**Reply by kjoetom** [reply]
Hi there,
in terms of transcribing in QualCoder: do you already know the VOSK open source offline speech recognition tool kit? You can find the documentation at the [Vosk Website](https://alphacephei.com/vosk/install) and the github project [here](https://github.com/alphacep/vosk-api).

This could be a good way to include the possibility to transcribe without limitations and also take care of data protection as this works offline.
In fact, there is already an audio editor here on github doing offline transcriptions using this toolkit (see [here](https://github.com/audapolis/audapolis)).

kjoe

**Reply by ccbogel** [reply, marked as answer]
Thanks for this information. I will add it to the manual.
